### ISSUE TITLE
Add 'watch <pluginname>' for plugin developers

### ISSUE THREAD

**Issue by SphinxC0re** [cuberite/cuberite#2709]
The execution of the command should lead to an inode watcher registration. I don't know if Windows has such a thing. Whenever the watcher triggers a 'change' event, the plugin should reload.

**Comment by madmaxoft** [comment]
I don't think this is a good idea. It doesn't work too well with the ZeroBrane Studio used for debugging, because it would abort the debug session whenever a file gets saved.

Also it wouldn't work too well with plugins that save their configuration to their own folder.

I vote against this feature.

**Comment by SphinxC0re** [comment]
The server wouldn't watch all the time. *You* could define yourself if you want a plugin to be watched

**Comment by madmaxoft** [comment]
If you really want this, it could be done via a plugin. Not exactly using `inotify`, simply check the folder every second using `cFile` functions
